recipes-extended/xen/files/0001-convert-block-tap-for-shared.patch

```diff
@@ -0,0 +1,551 @@
+From 164436bdff5620e605268ec82860728014dab7cb Mon Sep 17 00:00:00 2001
```

Comment:
Reminder for me to come back and finish reviewing this patch

Reply:
0001-convert-block-tap-for-shared.patch is introduced and fixed up in later commits.  I think they can and should all be squashed together, especially if we have xenmgr set TAPDISK3_CRYPTO_KEYDIR at the time of PR merging

Reply:
ok I'm done with this, resolving